WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -3646,3 +3667,23 @@ ZaAccountXFormView.prototype.getTabChoices =
 function() {
     return this.tabChoices;
 }
+
+ZaAccountXFormView.getEmailFromUid = 
+function(managerField) {
+	if (managerField) {
+		let email = '';
+		let managerLdapArray = managerField.split(',');
+		const usernameArray = managerLdapArray[0].split('uid=');
+		if (usernameArray.length !== 2) return '';
+		email = usernameArray[1];
+		managerLdapArray = managerLdapArray.filter(item => {
+			return item.includes('dc=');
+		});
+		managerLdapArray = managerLdapArray.map(item => {
+			const dcValueArray = item.split('=');
+			return dcValueArray.length === 2 ? item.split('=')[1] : '';
+		});
+		email = email.concat('@').concat(managerLdapArray.join('.'));
```

Comment:
I observed usage of two concat in single string.
Can't we use plus (+) operator?

Reply:
Fixed.